src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -334,9 +334,11 @@ def test_graph_group_scenario(self):
             return  # this test delete users which are beyond a SP's capacity, so quit...
 
         domain = username.split('@', 1)[1]
+        if domain == 'microsoft.com' and self.is_live:
```

Comment:
Actually, our live run CI uses an `@AzureSDKTeam.onmicrosoft.com` account and won't trigger this issue.

Reply:
You are right. I found this issue while fixing [another error](https://clitestresultstac.blob.core.windows.net/20201027-111238-748168/role.report.parallel.html) locally.